src/Model/Contact.php

```diff
@@ -2579,9 +2579,7 @@ public static function removeSharer($importer, $contact, array $datarray = [], $
 	public static function updateBirthdays()
 	{
 		$condition = [
-			'`bd` != ""
-			AND `bd` > "0001-01-01"
-			AND SUBSTRING(`bd`, 1, 4) != `bdyear`
+			'`bd` > "0001-01-01"
```

Comment:
Should we use one of our nifty constants?

Reply:
Done